tests/credential_plugins_test.py

```diff
@@ -137,13 +137,19 @@ class TestDelineaImports:
     library, so these tests are designed to fail if these wind up using the
     fallback import."""
 
-    def test_dsv_import(self):
-        from awx_plugins.credentials.dsv import SecretsVault  # noqa
+    def test_dsv_import(self) -> None:
+        from awx_plugins.credentials.dsv import SecretsVault  # noqa: F401
+
         # assert this module as opposed to older thycotic.secrets.vault
         assert SecretsVault.__module__ == 'delinea.secrets.vault'
 
-    def test_tss_import(self):
-        from awx_plugins.credentials.tss import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret  # noqa
+    def test_tss_import(self) -> None:
+        from awx_plugins.credentials.tss import (  # noqa: F401
+            DomainPasswordGrantAuthorizer,
+            PasswordGrantAuthorizer,
+            SecretServer,
+            ServerSecret,
+        )
```

Comment:
Same as above, let's move this out to keep the PR atomic.

Reply:
I have removed those codes and have reverted both of these comments back to # noqa.

Reply:
Looks like the revert was partial and needs to be iterated on: https://github.com/ansible/awx-plugins/pull/26#discussion_r1759175316.

Also, it's a good idea to put this change into a new PR and paste a link to that PR here.

Reply:
Thank you so much, that makes sense! I've done that over here: https://github.com/ansible/awx-plugins/pull/36

Reply:
Marking this as resolved.